server/container/util/src/main/java/org/apache/james/util/mime/MessageContentExtractor.java

```diff
@@ -220,6 +222,13 @@ public MessageContent merge(MessageContent fromInnerMultipart) {
                     htmlBody.map(Optional::of).orElse(fromInnerMultipart.getHtmlBody()));
         }
 
+        public Optional<String> mainTextContent(HtmlTextExtractor htmlTextExtractor) {
```

Comment:
I don't expect a `mainTextContent` to favor htmlBody over textBody but rather the opposite.

Reply:
can we have a good test coverage for that?

Reply:
> I don't expect a mainTextContent to favor htmlBody over textBody but rather the opposite.

Let's avoid behavioural changes.

Reply:
Then we need to rename the method at the very least.